### ISSUE TITLE
[Spree 2.1] Fix strong params isssue in /admin/enterprises/images_spec (1 spec)

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#4948]
spec/features/admin/enterprises/images_spec.rb:29

For some reason the preview image is not showing.

```
  2) Managing enterprise images as an Enterprise user images for an enterprise editing logo
     Failure/Error: expect(page).to have_selector(".image-field-group__preview-image[src*='#{file_name}']")
       expected to find visible css ".image-field-group__preview-image[src*='logo-white.png']" within #<Capybara::Node::Element tag="div" path="/html/body/div[1]/div[1]/div[4]/div/div/div/form/div[2]/div/div[3]/fieldset[8]/div[1]"> but there were no matches
     # ./spec/features/admin/enterprises/images_spec.rb:104:in `expect_preview_image'
     # ./spec/features/admin/enterprises/images_spec.rb:38:in `block (5 levels) in <top (required)>'
     # ./spec/features/admin/enterprises/images_spec.rb:37:in `block (4 levels) in <top (required)>'
```

**Comment by Matt-Yorkley** [comment]
@luisramos0 This is fixed by adding `:logo` to the list of permitted params in admin/enterprises_controller. I'm not sure what the plan is with the other `strong_params` PRs at the moment..?

**Comment by luisramos0** [comment]
ah, awesome, I can take care of that.
The plan for strong params: I am going to break the big PR in small PRs. I am going to do that tomorrow :+1:

**Comment by luisramos0** [comment]
done, added the fix to #4827 for now.
thanks!
